.github/workflows/kernel-demos.yml

```diff
@@ -126,6 +126,20 @@ jobs:
           submodules: 'recursive'
```

Comment:
Does this not fetch submodules?

Reply:
Fetching all the submodules takes a long time and delays the CI run significantly. I would strongly suggest fetching submodules one by one.